karma.conf.js

```diff
@@ -72,13 +73,21 @@ module.exports = config => {
     // enable / disable watching file and executing tests whenever any file changes
     autoWatch: true,
 
+    browserStack: {
+      username: process.env.BROWSERSTACK_USERNAME,
+      accessKey: process.env.BROWSERSTACK_ACCESS_KEY
+    },
+
     // start these browsers
     // available browser launchers: https://npmjs.org/browse/keyword/karma-launcher
-    browsers: ["ChromeHeadlessNoSandbox"],
+    browsers: ["bs_chrome_windows"],
```

Comment:
These changes will make it so that every time we run unit tests locally with `npm run test:unit`, `npm run test:unit:watch`, or `npm run test:unit:coverage`, it will run it on BrowserStack instead of locally. I believe by default these scripts should run the unit tests locally. Then we could have another npm script for running the unit tests on browserstack, similar to how we had `npm run test:unit:saucelabs`. Is there a reason you didn't follow the same approach we had?

Reply:
Added the script back to run unit tests using a separate karma config for browserstacks. I added the the browsers we previously for our unit test.